### PR TITLE
Development

### PR DESCRIPTION
**Pull request by mapra99**
Open Pull request to TSE Review #1 of Project: Apple Clone.

### CODE REVIEW

**Comment by mapra99** [comment]
.

File: index.html [hunk 1]
```diff
@@ -4,22 +4,88 @@
...
+                <nav class = "nav-header">
+                    <ul><!--
+                    --><li class = "nav-button nav-start"><a class = "anch" href="#"><i class="fab fa-apple"></i></a></li><!--
+                    --><li class = "nav-button"><a class = "anch" href="#">Store</a></li><!--
+                    --><li class = "nav-button"><a class = "anch"  href="#">Mac</a></li><!--
+                    --><li class = "nav-button"><a class = "anch"  href="#">iPod</a></li><!--
+                    --><li class = "nav-button"><a class = "anch"  href="#">iPhone</a></li><!--
+                    --><li class = "nav-button"><a class = "anch"  href="#">iPad</a></li><!--
+                    --><li class = "nav-button"><a class = "anch"  href="#">iTunes</a></li><!--
+                    --><li class = "nav-button"><a class = "anch"  href="#">Support</a></li><!--
```
Comment (bolah2009) [inline index.html:30]: Consider changing the navbar links to buttons.

**Review by bolah2009 — APPROVED** [review]
# Congratulations @mapra99  ! :tada:

You've done an awesome Job :muscle:

### Check out [this comment](https://github.com/mapra99/appleClone/pull/1#discussion_r305592356) before merging.

Good luck with your next project
Keep it up :star2:

Cheers
Bola

File: index.html (hunk 1 above)
Comment (mapra99) [reply index.html:30]: Thanks Bola. I considered the use of buttons in the navbar elements, but then I [changed the buttons to links](https://github.com/mapra99/appleClone/pull/1/commits/87cd5243fe23d91541a5783dca9aa8639f0b7844), following the suggestions made in [this](https://www.w3schools.com/howto/howto_js_topnav.asp) and [this](https://wearefieldtrip.com/journal/css-horizontal-navigation-bar/) article. Which alternative is preferable? Buttons or Links?

Comment (bolah2009) [reply index.html:30]: Those links are instructions on how to create a navbar. Using anchor element (`<a>`) is much easier and preferable in this context. Buttons are mostly used in forms or for call-to-action components. According to Microverse guidelines, you are expected to use buttons but it's okay to use anchor element (`<a>`).

**Pull request merged**